packages/cardhost/app/components/cardhost-right-edge.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+
+const typeToInputTypeMap = {
+  '@cardstack/core-types::string': 'Text Field',
+  '@cardstack/core-types::case-insensitive': 'Text Field',
+  '@cardstack/core-types::boolean': 'Checkbox',
+  '@cardstack/core-types::date': 'Date',
+  '@cardstack/core-types::integer': 'Text Field',
+}
+
+export default class CardhostRightEdge extends Component {
```

Comment:
There is a good chance that these components will move into the core package--the "cardhost" package is meant to be a lab for developing things. Because of that, let's not prefix this component with the name of "Cardhost"